packages/xrpl/src/Wallet/fundWallet.ts

```diff
@@ -112,6 +115,8 @@ async function fundWallet(
       JSON.stringify({
         destination: walletToFund.classicAddress,
         xrpAmount: options?.amount,
+        userAgent: 'xrpl.js',
+        useCase: options?.useCase,
```

Comment:
I'm still not a big fan of the name `useCase` - it's not very descriptive for someone who's looking at this library and trying to figure out how they are supposed to use it.

Reply:
what would you name it instead? 
also I did add an explanation of what useCase is in the function params

Reply:
I think useCase is the best I can think of